ReleaseChecklist.md

```diff
@@ -22,7 +22,7 @@
  - [ ] Create a pull request from ``develop`` to ``release``, wait for the tests, then merge it.
  - [ ] Make a final check that there are no platform-dependency issues in the ``solidity-test-bytecode`` repository.
  - [ ] Wait for the tests for the commit on ``release``, create a release in Github, creating the tag (click the `PUBLISH RELEASE` button on the release page.)
- - [ ] Wait for the CI runs on the tag itself (travis and appveyor should push artifacts onto the Github release page).
+ - [ ] Wait for the CI runs on the tag itself (travis should push artifacts onto the Github release page).
  - [ ] Run ``scripts/create_source_tarball.sh`` while being on the tag to create the source tarball. Make sure to create ``prerelease.txt`` before: (``echo -n > prerelease.txt``). This will create the tarball in a directory called ``upload``.
  - [ ] Take the tarball from the upload directory (its name should be ``solidity_x.x.x.tar.gz``, otherwise ``prerelease.txt`` was missing in the step before) and upload the source tarball to the release page.
 
```

Comment:
"copy solc-windows.zip from the release page..." below has to be removed, instead something similar to what is done for MacOS should be added.

Reply:
Ah sorry, no, nothing has to be removed, only something has to be added.